package.json

```
@@ -9,10 +9,10 @@
     "scripts": {
         "images": "imagemin '_site/assets/images' --out-dir='_site/assets/images'",
         "install:playground": "cd src/playground && npm install --no-package-lock",
-        "watch:sass": "node-sass --watch src/assets/scss -o src/assets/css",
+        "watch:sass": "sass --style=compressed --no-source-map --watch src/assets/scss:src/assets/css",
```

```suggestion
        "watch:sass": "sass --watch src/assets/scss:src/assets/css",
```

or we can do both :)